OSRMTextInstructions/OSRMTextInstructions.swift

```diff
@@ -406,7 +406,7 @@ extension NSAttributedString: Tokenized {
             exitOrdinal = ordinalFormatter.string(from: exitIndex as NSNumber)!
         }
         let modifierConstants = constants["modifier"] as! [String: String]
-        let modifierConstant = modifierConstants[modifier ?? "straight"]!
+        let modifierConstant = modifierConstants[modifier == "none" ? "straight" : modifier]!
```

Comment:
`none` is not a constant on the js side of things. Is this the optimal solution here @1ec5 @frederoni?

Reply:
Yes, this is correct. To be clear, this is only necessary because we’re getting the `description` of the maneuver above; otherwise, we’d compare to `.none`.